examples/vue-router/App.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div>
+    <nav>
+      <router-link class="nav-route" :to="{ name: '/pages/about' }">
```

Comment:
Does `<router-link to="/about">About</router-link>` also work?

Reply:
yes